src/lesson10/hw/Pet.java

```diff
@@ -0,0 +1,24 @@
+package lesson10.hw;
+//I create a class Pet, which is a child class of the Animal class.
+// When its constructor is called, it should print "I am Pet" on the screen,
+// and the variable "eyes" should be assigned the value 2.
+// Additionally, in the Pet class, I add variables "name" and "tail", where "tail" should equal 1.
+// Also, I create a field called "paw", which will be equal to 4.
+// I create a method "run", which will print the message "Pet runs" on the screen,
+// and a method "jump", which will print "Pet jumps".
+public class Pet extends Animal{
+    String name;
+    final int TAIL = 1;
+    final int PAW = 4;
+    public Pet(String name) {
+        this.eyes = 2;
```

Comment:
Create in class Animal costructor with parameter "name" and call here super(2)
What is happened with parameter "name" ?
Run app